src/TailwindCssPreset.php

```diff
@@ -30,10 +30,10 @@ public static function installAuth()
     protected static function updatePackageArray(array $packages)
     {
         return array_merge([
-            'laravel-mix' => '^4.0.14',
+            'laravel-mix' => '^5.0',
             'laravel-mix-purgecss' => '^4.1',
             'laravel-mix-tailwind' => '^0.1.0',
-            'tailwindcss' => '^1.0',
```

Comment:
`^1.0` will already install 1.2 now, or 1.3 later, or whatever. There's no need to change this constraint here.

Reply:
You're right! Sorry, my bad. Reverted that in an update.